docs/source/rules.rst

```diff
@@ -161,53 +159,67 @@ Example use of that function within a rule::
 Rules Testing
 -------------
 
-In order to test the effectiveness of our new rules, you can run a set of local integration tests to verify alerts would be triggered.  The `stream_alert_cli.py` command line tool comes built-in with a `lambda test` command.
+In order to test the effectiveness of new rules, local integration tests can be ran to verify that alerts would be triggered given a certain input.  The `stream_alert_cli.py` command line tool comes built-in with a `lambda test` command which does exactly this.
 
 Configuration
 ~~~~~~~~~~~~~
 
-To get started, create (or find) an example log for your given rule.  If the rule you added expects incoming records to be JSON, add a raw JSON record into the ``trigger_events.json`` file for the related stream.
+To test a new rule, first create a new file under `test/integration/rules` named `rule_name.json`.  This file should contain this exact structure::
+  
+  {
+    "records": [
+      {
+        "data": {} or "",
+        "description": "of the test",
+        "trigger": true or false,
+        "source": "kinesis_stream_name" or "s3_bucket_id",
+        "service": "kinesis" or "s3"
+      }
+    ]
+  }
 
-Example logs will be stored in the ``test/integration/fixtures/kinesis`` subdirectory.  A new folder should be created for each Kinesis stream as declared in your `sources.json <conf-datasources.html>`_.
+The `data` key can either be a Map or a String.  Normally all json types should be in Map format while others should be String (csv, kv, syslog).
 
-Within each of these folders, add the following two files:
+The `description` key should be a short sentence describing the intent of the test.
 
-* ``trigger_events.json``
-* ``non_trigger_events.json``
+The `trigger` key indicates whether or not this record should produce an alert or not.  This is used to determine the overall coverage of this rule.  Coverage indicates that all of our test records produce the correct amount of alerts.
```

Comment:
The `trigger` key indicates whether a record should produce an alert or not.  I'm still not a fan of coverage since that is commonly used to refer to "code coverage"